src/API/Google/AssetFieldType.php

```diff
@@ -0,0 +1,153 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Google;
+
+use Google\Ads\GoogleAds\V11\Enums\AssetFieldTypeEnum\AssetFieldType as AdsAssetFieldType;
+use Automattic\WooCommerce\GoogleListingsAndAds\Internal\StatusMapping;
+use UnexpectedValueException;
+
+
+/**
+ * Mapping between Google and internal AssetFieldTypes
+ * https://developers.google.com/google-ads/api/reference/rpc/v11/AssetFieldTypeEnum.AssetFieldType
+ *
+ * @since x.x.x
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Google
+ */
+class AssetFieldType extends StatusMapping {
```

Comment:
💅  WDYT about renaming this to `AssetFieldTypeMapping`?

Reply:
The other classes that are implementing `StatusMapping` do not use the suffix Mapping. See: 

- https://github.com/woocommerce/google-listings-and-ads/blob/0ffb057cbe73a3b1989bb35babed13899da1c4cf/src/API/Google/CampaignType.php#L17
- https://github.com/woocommerce/google-listings-and-ads/blob/0ffb057cbe73a3b1989bb35babed13899da1c4cf/src/API/Google/BillingSetupStatus.php#L15

So I think it is ok if we leave as it is.